models/get-models.js

```diff
@@ -16,3 +16,28 @@ exports.selectArticle = (article_id) => {
       return rows[0];
     });
 };
+
+exports.selectArticles = () => {
+  return db
+    .query("SELECT * FROM comments")
+    .then(({ rows }) => {
+      const commentData = {};
+
+      for (let i = 0; i < rows.length; i++) {
+        const currentArticle_id = rows[i].article_id;
+        commentData[currentArticle_id] =
+          commentData[currentArticle_id] + 1 || 1;
+      }
+      return commentData;
+    })
+    .then((commentData) => {
+      return db.query("SELECT * FROM articles").then(({ rows }) => {
```

Comment:
Is there a way we can do this with only one query? Remember we're working with relational databases and we have the JOIN keyword to take advantage of